src/main/rx/rx.c

```diff
@@ -419,6 +420,14 @@ bool rxUpdateCheck(timeUs_t currentTimeUs, timeDelta_t currentDeltaTime)
             rxIsInFailsafeMode = (frameStatus & RX_FRAME_FAILSAFE) != 0;
             rxSignalReceived = !rxIsInFailsafeMode;
             needRxSignalBefore = currentTimeUs + needRxSignalMaxDelayUs;
+
+            if (frameStatus & RX_FRAME_DROPPED) {
+            	// No (0%) signal
+            	setRssiUnfiltered(0, RSSI_SOURCE_RX_PROTOCOL);
```

Comment:
Using `RSSI_SOURCE_RX_PROTOCOL` is wrong. `RSSI_SOURCE_RX_PROTOCOL` is used for RX protocols that send real RSSI values (currently only FPort). They should not be overridden with emulated RSSI values. Use a new `RSSI_SOURCE_FRAME_ERRORS` (or similar) instead.